src/layouts/Main/Main.js

```diff
@@ -47,15 +48,15 @@ const Main = ({
     threshold: 38,
   });
 
-  let pageNavColorRegex = new RegExp(/mindshare|authors|blog/gi);
+  let pageNavColorRegex = new RegExp(/mindshare|article/gi);
 
   return (
     <Box>
       <Box bgcolor={bgcolor} position={'relative'} zIndex={theme.zIndex.appBar}>
         <Container paddingTop={'8px !important'} paddingBottom={'0 !important'}>
           <TopNav
             colorInvert={
-              url?.match(pageNavColorRegex) !== null ? true : colorInvert
+              model?.match(pageNavColorRegex) !== null ? true : colorInvert
```

Comment:
@darwin808 I changed this regex check to look at the model, so now, only the midnshare (blog home) and article content model reverse

Reply:
noted on this